bittide-instances/src/Bittide/Instances/Hitl/Driver/VexRiscvTcp.hs

```diff
@@ -234,10 +238,10 @@ case_testTcpClient = do
       projectDir <- findParentContaining "cabal.project"
       let
         hitlDir = projectDir </> "_build" </> "hitl"
-        stdoutLog = hitlDir </> "picocom-stdout.log"
-        stderrLog = hitlDir </> "picocom-stderr.log"
-      putStrLn $ "logging stdout to `" <> stdoutLog <> "`"
-      putStrLn $ "logging stderr to `" <> stderrLog <> "`"
+        picoOutLog = hitlDir </> "picocom-stdout.log"
+        picoErrLog = hitlDir </> "picocom-stderr.log"
+      putStrLn $ "logging stdout to `" <> picoOutLog <> "`"
+      putStrLn $ "logging stderr to `" <> picoErrLog <> "`"
```

Comment:
You're patching commented code here.
Do you known why we even have this code here in comments?

Reply:
It's old code, I've removed it